### PR TITLE
Update the debug document for tizen rt-ide.

### PR DESCRIPTION
**Pull request by vivek-ellur**
Signed-off-by: vivek ellur <vivek.ellur@samsung.com>

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/application/tizen-studio/rt-ide/getting-started/debug.md [hunk 1]
```diff
@@ -28,23 +28,59 @@ To debug the application:
...
-   ![Debug perspective](./media/rt_debug_perspective.png)
-
-   When an action is executed on the board and it passes through a break point, the line is in the suspended state (see the figure above).
-
-   You can perform various debugging functions (such as resume, step into, and step over) by using the toolbar in the **Debug** perspective.
+5. Before the debugging starts, the Confirm Perspective Switch dialog is shown. Click **No** and add **Source Mapping** before proceeding to debug perspective.
+
+   ![Debug perspectiveno](./media/rt_debug_confirmno.png)
+   
+6. To add **Source Mapping**, click **Debug Configurations** in the debug dropdown.
```
Comment (sabeena21) [inline docs/application/tizen-studio/rt-ide/getting-started/debug.md]: debug dropdown> debug drop-down list

File: docs/application/tizen-studio/rt-ide/getting-started/debug.md [hunk 2]
```diff
@@ -28,23 +28,59 @@ To debug the application:
...
+
+   ![Debug configurations](./media/rt_debug_config_menu.png)
+
+7. Expand the **GDB OpenOCD Debugging** from the list in the left and select the available configuration. Select the **Source** tab and click **Add** button.
```
Comment (sabeena21) [inline docs/application/tizen-studio/rt-ide/getting-started/debug.md]: click **Add** .
It is not required to add the word button

File: docs/application/tizen-studio/rt-ide/getting-started/debug.md [hunk 3]
```diff
@@ -28,23 +28,59 @@ To debug the application:
...
+
+	![Debug sourceadd](./media/rt_debug_source.png)
+   
+8. Select **Path Mapping** option from the **Add Source** dialog.
```
Comment (sabeena21) [inline docs/application/tizen-studio/rt-ide/getting-started/debug.md]: Select **Path Mapping** option from the **Add Source** dialog box.

File: docs/application/tizen-studio/rt-ide/getting-started/debug.md [hunk 4]
```diff
@@ -28,23 +28,59 @@ To debug the application:
...
+
+	![Debug sourcepath](./media/rt_debug_pathadd.png)
+	
+9. In the **Path Mapping** dialog click **Add** button and enter the name for new path mapping.
```
Comment (sabeena21) [inline docs/application/tizen-studio/rt-ide/getting-started/debug.md]: In the **Path Mapping** dialog box, click **Add** , and enter the name for new path mapping.

File: docs/application/tizen-studio/rt-ide/getting-started/debug.md [hunk 5]
```diff
@@ -28,23 +28,59 @@ To debug the application:
...
+	
+	- In the Configuration Path, Add "/root/tizenrt" as path.
```
Comment (sabeena21) [inline docs/application/tizen-studio/rt-ide/getting-started/debug.md]: "/root/tizenrt" > `/root/tizenrt`

File: docs/application/tizen-studio/rt-ide/getting-started/debug.md [hunk 6]
```diff
@@ -28,23 +28,59 @@ To debug the application:
...
+	- In the Local System FilePath, Select the path where the TizenRT platform code is present.
```
Comment (sabeena21) [inline docs/application/tizen-studio/rt-ide/getting-started/debug.md]: Local System FilePath> Local file system path

File: docs/application/tizen-studio/rt-ide/getting-started/debug.md [hunk 7]
```diff
@@ -28,23 +28,59 @@ To debug the application:
...
+	
+	![Debug sourcepath](./media/rt_debug_pathmapping.png)
+	
+	![Debug filepath](./media/rt_debug_filepath.png)
+	
+	![Debug rtcodepath](./media/rt_debug_fileaddpath.png)
+	
+10. Click **OK** button after the TizenRT platform code has been selected.
```
Comment (sabeena21) [inline docs/application/tizen-studio/rt-ide/getting-started/debug.md]: Please remove the word "button"

File: docs/application/tizen-studio/rt-ide/getting-started/debug.md [hunk 8]
```diff
@@ -28,23 +28,59 @@ To debug the application:
...
+	
+	![Debug filepathok](./media/rt_debug_filepathok.png)
+	
+	Once you have added the path mapping, you can see the new path mapping entry in **source** tab.
```
Comment (sabeena21) [inline docs/application/tizen-studio/rt-ide/getting-started/debug.md]: Please change "once" with After"

After,  you have added the path mapping, you can see the new path mapping entry in **source** tab.

File: docs/application/tizen-studio/rt-ide/getting-started/debug.md [hunk 9]
```diff
@@ -28,23 +28,59 @@ To debug the application:
...
+	
+	![Debug pathmapadded](./media/rt_debug_pathmapadded.png)
+	
+	After path mapping has been added, Click **Debug** button to start debugging.
```
Comment (sabeena21) [inline docs/application/tizen-studio/rt-ide/getting-started/debug.md]: remove "button"

File: docs/application/tizen-studio/rt-ide/getting-started/debug.md [hunk 10]
```diff
@@ -28,23 +28,59 @@ To debug the application:
...
+	Confirm Perspective dialog is shown, To switch to the **Debug** perspective, click **Yes**.
```
Comment (sabeena21) [inline docs/application/tizen-studio/rt-ide/getting-started/debug.md]: Confirm Perspective dialog box is show, to switch to the **Debug** perspective, click **Yes**.

Comment (mijin85cho) [reply docs/application/tizen-studio/rt-ide/getting-started/debug.md]: Hello, @sabeena21 
I'm not sure whether this "A is show" is right sentence to use here.

How about using other words such as "Confirm Perspective dialog box pops up,"?
Please check.

**Comment by mijin85cho** [comment]
Hello, @vivek-ellur
Would you please check the review comments?

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/446

**Comment by mijin85cho** [comment]
Thanks for the review and update.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**